<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>848</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="AddButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="EditButton">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>10</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Bahnschrift Light Condensed</family>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="PauseButton">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>10</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Impact</family>
      <pointsize>25</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>I I</string>
    </property>
   </widget>
   <widget class="QLabel" name="Pause">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>350</y>
      <width>250</width>
      <height>250</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(Img/Pause.png);
z-index:3;</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="ShadowPause">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>70</y>
      <width>301</width>
      <height>781</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgba(0,0,0,0.5);
border-radius:90px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="SwitchStyles">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>10</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>301</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:5px; background-color:rgba(255,255,255,0.6);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <zorder>label</zorder>
   <zorder>AddButton</zorder>
   <zorder>EditButton</zorder>
   <zorder>PauseButton</zorder>
   <zorder>Pause</zorder>
   <zorder>ShadowPause</zorder>
   <zorder>SwitchStyles</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
